opendmarc/opendmarc.conf.sample

```diff
@@ -1,6 +1,7 @@
 ## opendmarc.conf -- configuration file for OpenDMARC filter
 ##
-## Copyright (c) 2012-2015, The Trusted Domain Project.  All rights reserved.
+## Copyright (c) 2012-2015, 2018, The Trusted Domain Project.
```

Comment:
You don't need to do these yourself.  There's a pre-release script I run that will do these for you.